httpbakery/client.go

```diff
@@ -271,7 +273,7 @@ func (ctxt *clientContext) obtainThirdPartyDischarge(originalLocation string, ca
 	if err == nil {
 		return resp.Macaroon, nil
 	}
-	log.Printf("discharge post got error %#v", err)
+	logger.Debugf("discharge post got error %#v", err)
```

Comment:
We should probably just delete the log message - the error will be returned and should be logged anyway by our caller.